paas-redirect/app/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
this file is kinda strange. Does it still work if we make this an empty file?

Reply:
Yes it does - it's empty now 👍